shopify_demo.world

```diff
@@ -0,0 +1 @@
"extendType":"image"},{"url":"https://cdn.shopify.com/s/files/1/0549/6399/4647/products/headphones.jpg?v=1636658742","mimeType":"image/png","format":"png","extendType":"image"},{"url":"https://model3d.shopifycdn.com/models/o/9b220a1a1e22e637/Headphones_Pink.glb","mimeType":"model/gltf-binary","format":"glb","filesize":941852,"extendType":"model"},{"url":"https://model3d.shopifycdn.com/models/o/4c676cca63154c94/Headphones_White.glb","mimeType":"model/gltf-binary","format":"glb","filesize":951492,"extendType":"model"},{"url":"https://model3d.shopifycdn.com/models/o/d678bc2e5d597b8c/Headphones_Black.glb","mimeType":"model/gltf-binary","format":"glb","filesize":1379104,"extendType":"model"}]}],"shopifyDomain":"https://xrengine-test-site.myshopify.com/","shopifyToken":"878e6efb8645b374a57b0439ac204ad3","shopifyProductId":"Z2lkOi8vc2hvcGlmeS9Qcm9kdWN0LzY2MTAxNTI1ODcyODc=","shopifyProductItemId":3,"shopifyProductItems":[{"value":0,"label":"Image 1","media":{"url":"https://cdn.shopify.com/s/files/1/0549/6399/4647/products/image_1.jpg?v=1636577558","mimeType":"image/png","format":"png","extendType":"image"}},{"value":1,"label":"Image 2","media":{"url":"https://cdn.shopify.com/s/files/1/0549/6399/4647/products/image.jpg?v=1636577558","mimeType":"image/png","format":"png","extendType":"image"}},{"value":2,"label":"Video 1","media":{"url":"https://cdn.shopify.com/videos/c/vp/5cb590daf49b40e19f3f19ec900c2df6/5cb590daf49b40e19f3f19ec900c2df6.m3u8","mimeType":"application/x-mpegURL","format":"m3u8","height":720,"width":1280,"extendType":"video"}},{"value":3,"label":"Model 1","media":{"url":"https://model3d.shopifycdn.com/models/o/baea3e723e1ba676/razer_laptop.glb","mimeType":"model/gltf-binary","format":"glb","filesize":1101196,"extendType":"model"}}],"extendType":"model","extend":{"src":"https://model3d.shopifycdn.com/models/o/baea3e723e1ba676/razer_laptop.glb","textureOverride":"","castShadow":false,"receiveShadow":false,"loop-animation":{"activeClipIndex":-1,"hasAvatarAnimations":false}}}},{"name":"interact","props":{"interactable":true,"interactionType":"infoBox","interactionText":"ThinkPad Laptop X1","interactionThemeIndex":0,"interactionName":"ThinkPad Laptop X1","interactionDescription":"With the Intel® Evo™ platform, the ThinkPad X1 Carbon Gen 9 laptop delivers a powerhouse combination of performance, responsiveness, battery life, and stunning visuals. With up to 11th Gen Intel® Core™ i7 vPro® processors, you can count on an exceptional experience, anywhere.","interactionImages":["https://cdn.shopify.com/s/files/1/0549/6399/4647/products/image_1.jpg?v=1636577558","https://cdn.shopify.com/s/files/1/0549/6399/4647/products/image.jpg?v=1636577558"],"interactionVideos":["https://cdn.shopify.com/videos/c/vp/5cb590daf49b40e19f3f19ec900c2df6/5cb590daf49b40e19f3f19ec900c2df6.m3u8"],"interactionUrls":[],"interactionModels":["https://model3d.shopifycdn.com/models/o/baea3e723e1ba676/razer_laptop.glb"]}}],"parent":"2266BED7-6CC4-48A6-95DD-9BCD3CF9EAFC","index":7}}}
```

Comment:
this can be moved to a new custom scene in /packages/projects/default-project/shopify-demo.scene.json
it will then be available directly from the editor